src/rockstor/fs/btrfs.py

```diff
@@ -782,346 +809,25 @@ def device_scan():
 
 def btrfs_uuid(disk):
     """return uuid of a btrfs filesystem"""
-    o, e, rc = run_command([BTRFS, 'filesystem', 'show', '/dev/%s' % disk])
+    o, e, rc = run_command(
+        [BTRFS, 'filesystem', 'show', '/dev/disk/by-id/%s' % disk])
     return o[0].split()[3]
 
 
 def btrfs_label(disk):
-    o, e, rc = run_command([BTRFS, 'filesystem', 'label', '/dev/%s' % disk])
+    o, e, rc = run_command(
+        [BTRFS, 'filesystem', 'label', '/dev/disk/by-id/%s' % disk])
     return o[0]
 
 
 def btrfs_importable(disk):
```

Comment:
even though this is not a function you introduced, it's misleading and not used. Remove it if you can please.